tests/test_gate_seal.py

```diff
@@ -428,3 +428,46 @@ def test_cannot_seal_twice_in_one_tx(gate_seal, sealables, sealing_committee):
     gate_seal.seal(sealables, sender=sealing_committee)
     with reverts("gate seal: expired"):
         gate_seal.seal(sealables, sender=sealing_committee)
+
+
+def test_raw_call_success_should_be_false_when_sealable_reverts_on_pause(project, deployer, generate_sealables, sealing_committee, seal_duration_seconds, expiry_timestamp):
+    """
+        `raw_call` without `max_outsize` and with `revert_on_failure=True` for some reason returns the boolean value of storage[0] :^)
```

Comment:
not `storage[0]`, should be `memory[0]` (it's `MLOAD`, not `SLOAD` that the compiler generates extra)